release-notes/opensearch-dashboards-maps.release-notes-2.3.0.0.md

```diff
@@ -0,0 +1,8 @@
+## Version 2.3.0.0 Release Notes
+Compatible with OpenSearch and OpenSearch Dashboards Version 2.3.0
+
+### Enhancements
+* Adds integration tests in the repo for customImportMap plugin ([#30](https://github.com/opensearch-project/dashboards-maps/pull/30))
+
+### Refactoring
+* Adds dashboards as suffix to plugin name ([#27](https://github.com/opensearch-project/dashboards-maps/pull/27))
```

Comment:
#27 is tagged and merged to 2.2, do we still need to add it to 2.3 release note? I know it was missed on 2.2 release note since it was a late fixing PR...

Reply:
I didn't know that. I checked that this PR was not present in older release notes hence I added it. Let me remote this